src/main/scala/org/broadinstitute/dsde/firecloud/model/Profile.scala

```diff
@@ -132,6 +132,7 @@ trait mappedPropVals {
         f.get(this) match {
           case x: String => f.getName -> x
           case y: Option[_] => f.getName -> y.asInstanceOf[Option[_]].getOrElse("").toString
+          case z => f.getName -> z.toString
```

Comment:
That would catch Option too which is bad. You'd get "Some(whatever)" if you did.